src/trans/sedrenames.txt

```diff
@@ -0,0 +1,147 @@
+s/DIR_TRANSAD( *($|\(| |\*))/DIR_TRANSAD_VARIANTDESIGNATOR\1/g
+s/DIST_GRID_32_CTL_MOD/DIST_GRID_32_CTL_MOD_VARIANTDESIGNATOR/g
+s/dist_grid_32( *($|\(| |\*))/dist_grid_32_VARIANTDESIGNATOR\1/g
+s/DIST_GRID_32( *($|\(| |\*))/DIST_GRID_32_VARIANTDESIGNATOR\1/g
+s/DIST_GRID_CTL_MOD/DIST_GRID_CTL_MOD_VARIANTDESIGNATOR/g
+s/dist_grid( *($|\(| |\*))/dist_grid_VARIANTDESIGNATOR\1/g
+s/DIST_GRID( *($|\(| |\*))/DIST_GRID_VARIANTDESIGNATOR\1/g
+s/DIST_SPEC_CONTROL_MOD/DIST_SPEC_CONTROL_MOD_VARIANTDESIGNATOR/g
+s/dist_spec( *($|\(| |\*))/dist_spec_VARIANTDESIGNATOR\1/g
+s/DIST_SPEC( *($|\(| |\*))/DIST_SPEC_VARIANTDESIGNATOR\1/g
+s/ectrans_mod/ectrans_mod_VARIANTDESIGNATOR/g
+s/FFTB_PLAN/FFTB_PLAN_VARIANTDESIGNATOR/g
+s/FFTB_TYPE/FFTB_TYPE_VARIANTDESIGNATOR/g
+s/FIELD_SPLIT_MOD/FIELD_SPLIT_MOD_VARIANTDESIGNATOR/g
+s/FOURIER_IN_MOD/FOURIER_IN_MOD_VARIANTDESIGNATOR/g
+s/FOURIER_INAD_MOD/FOURIER_INAD_MOD_VARIANTDESIGNATOR/g
+s/FOURIER_OUT_MOD/FOURIER_OUT_MOD_VARIANTDESIGNATOR/g
+s/FOURIER_OUTAD_MOD/FOURIER_OUTAD_MOD_VARIANTDESIGNATOR/g
+s/FSC_MOD/FSC_MOD_VARIANTDESIGNATOR/g
+s/FSCAD_MOD/FSCAD_MOD_VARIANTDESIGNATOR/g
+s/FSPGL_INT_MOD/FSPGL_INT_MOD_VARIANTDESIGNATOR/g
+s/FTDIR_CTL_MOD/FTDIR_CTL_MOD_VARIANTDESIGNATOR/g
+s/FTDIR_CTLAD_MOD/FTDIR_CTLAD_MOD_VARIANTDESIGNATOR/g
+s/FTDIR_MOD/FTDIR_MOD_VARIANTDESIGNATOR/g
+s/FTDIRAD_MOD/FTDIRAD_MOD_VARIANTDESIGNATOR/g
+s/FTINV_CTL_MOD/FTINV_CTL_MOD_VARIANTDESIGNATOR/g
+s/FTINV_CTLAD_MOD/FTINV_CTLAD_MOD_VARIANTDESIGNATOR/g
+s/FTINV_MOD/FTINV_MOD_VARIANTDESIGNATOR/g
+s/FTINVAD_MOD/FTINVAD_MOD_VARIANTDESIGNATOR/g
+s/GATH_GRID_32_CTL_MOD/GATH_GRID_32_CTL_MOD_VARIANTDESIGNATOR/g
+s/gath_grid_32( *($|\(| |\*))/gath_grid_32_VARIANTDESIGNATOR\1/g
+s/GATH_GRID_32( *($|\(| |\*))/GATH_GRID_32_VARIANTDESIGNATOR\1/g
+s/GATH_GRID_CTL_MOD/GATH_GRID_CTL_MOD_VARIANTDESIGNATOR/g
+s/gath_grid( *($|\(| |\*))/gath_grid_VARIANTDESIGNATOR\1/g
+s/GATH_GRID( *($|\(| |\*))/GATH_GRID_VARIANTDESIGNATOR\1/g
+s/GATH_SPEC_CONTROL_MOD/GATH_SPEC_CONTROL_MOD_VARIANTDESIGNATOR/g
+s/gath_spec( *($|\(| |\*))/gath_spec_VARIANTDESIGNATOR\1/g
+s/GATH_SPEC( *($|\(| |\*))/GATH_SPEC_VARIANTDESIGNATOR\1/g
+s/GPNORM_TRANS_CTL_MOD/GPNORM_TRANS_CTL_MOD_VARIANTDESIGNATOR/g
+s/gpnorm_trans( *($|\(| |\*))/gpnorm_trans_VARIANTDESIGNATOR\1/g
+s/GPNORM_TRANS( *($|\(| |\*))/GPNORM_TRANS_VARIANTDESIGNATOR\1/g
+s/ini_spec_dist( *($|\(| |\*))/ini_spec_dist_VARIANTDESIGNATOR\1/g
+s/INI_SPEC_DIST/INI_SPEC_DIST_VARIANTDESIGNATOR/g
+s/INIGPTR_MOD/INIGPTR_MOD_VARIANTDESIGNATOR/g
+s/INTERPOL_DECOMP_MOD/INTERPOL_DECOMP_MOD_VARIANTDESIGNATOR/g
+s/INV_TRANS_CTL_MOD/INV_TRANS_CTL_MOD_VARIANTDESIGNATOR/g
+s/INV_TRANS_CTLAD_MOD/INV_TRANS_CTLAD_MOD_VARIANTDESIGNATOR/g
+s/inv_trans( *($|\(| |\*))/inv_trans_VARIANTDESIGNATOR\1/g
+s/INV_TRANS( *($|\(| |\*))/INV_TRANS_VARIANTDESIGNATOR\1/g
+s/inv_transad( *($|\(| |\*))/inv_transad_VARIANTDESIGNATOR\1/g
+s/INV_TRANSAD/INV_TRANSAD_VARIANTDESIGNATOR/g
+s/jprb/TYPEDESIGNATOR_LOWER/g
+s/JPRB/TYPEDESIGNATOR_UPPER/g
+s/JPRH/JPRD/g
```

Comment:
This line confuses me a bit. With this sed, [this line](https://github.com/ecmwf-ifs/ectrans/blob/develop/src/trans/internal/suleg_mod.F90#L20C1-L20C35):
```
USE PARKIND2          ,ONLY : JPRH
```
will become this line
```
USE PARKIND2          ,ONLY : JPRD
```
which should not compile because `PARKIND2` does not contain `JPRD`. What am I missing, question for @piotrows I guess?

Reply:
And as before, we need to decide first if we are happy to give up on the option for extended precision. The `REALHUGE` Fiat CPP definition appears to be defunct, so I guess we can go ahead.

Reply:
> This line confuses me a bit. With this sed, [this line](https://github.com/ecmwf-ifs/ectrans/blob/develop/src/trans/internal/suleg_mod.F90#L20C1-L20C35):
> 
> ```
> 
> USE PARKIND2          ,ONLY : JPRH
> 
> ```
> 
> will become this line
> 
> ```
> 
> USE PARKIND2          ,ONLY : JPRD
> 
> ```
> 
> which should not compile because `PARKIND2` does not contain `JPRD`. What am I missing, question for @piotrows I guess?

It works because SED will replace both PARKIND1 and PARKIND2 with EC_PARKIND

Reply:
Oh I see, gotcha.